src/main/scala/pbdirect/FieldIndex.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2019 Beyond the lines
```

Comment:
@juanpedromoreno What do you think we should do about the copyright headers in this project? I am not a lawyer but it seems strange to add this copyright to newly added files.

Reply:
Should we change it for something like this?
https://github.com/ChristopherDavenport/circuit/blob/master/core/src/main/scala/io/chrisdavenport/circuit/CircuitBreaker.scala#L2

Reply:
Yeah, that sounds reasonable to me.

We can tweak the sbt-headers accordingly to make the copyright the more accurate possible.

Reply:
I've been looking into the MIT licence a little.

* Apparently there is no need for a copyright header in every file. The Apache Foundation recommend it if you are using Apache 2.0, but there is no need for MIT.
* There is no need to list changes. The example @fedefernandez linked above is from a project using Apache 2.0, which has a requirement to list changes.
* The only requirement is a `LICENSE` file containing a copyright notice and the MIT licence text.

So the simplest solution could be:
* remove the source file headers
* add a note to the `LICENSE` file saying 47 Degrees forked the project in 2019 and we claim copyright for any code added or changed since the fork.

What do you think? @juanpedromoreno @fedefernandez 

References:
* https://choosealicense.com/licenses/mit/
* [advice from some random SO user](https://softwareengineering.stackexchange.com/a/277699)

Reply:
Sounds good to me

Reply:
Sounds good to me too 👍

Reply:
Cool, I'll do that in a separate PR after I've merged this.